opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -701,16 +701,30 @@ void Schedule::iterateScheduleSection(std::size_t load_start, std::size_t load_e
     void Schedule::shut_well(const std::string& well_name, std::size_t report_step) {
         this->internalWELLSTATUSACTIONXFromPYACTION(well_name, report_step, "SHUT");
     }
+    void Schedule::shut_well(const std::string& well_name) {
+        this->internalWELLSTATUSACTIONXFromPYACTION(well_name, this->current_report_step, "SHUT");
+    }
```

Comment:
This has a tiny bit of duplication&ndash;i.e., the string `"SHUT"`.  Would it make sense to implement the function as
```C++
void Schedule::shut_well(const std::string& well_name) {
    this->shut_well(well_name, this->current_report_step);
}
```
instead, to remove that source of **potential** inconsistency?  Similar for the other `*_well()` functions too.